app/services/backup.tsx

```diff
@@ -105,31 +106,43 @@ class Backup {
                     console.log('here log itemnew',itemnew)
                      const childData: any = [];
                      childData.push(itemnew);
-                     await userRealmCommon.create<ChildEntity>(ChildEntitySchema, childData);
+                    //await userRealmCommon.create<ChildEntity>(ChildEntitySchema, childData);
+                    return userRealmCommon.create<ChildEntity>(ChildEntitySchema, childData).then(function(results){
+                        console.log("results>>", results);
+                        return results
+                     })
+                     
                 }
             });
+ 
             const notiFlagObj = { key: 'generateNotifications', value: true };
             dispatch(setInfoModalOpened(notiFlagObj));
+ 
             await Promise.all(resolvedPromises).then(async (item:any) => {
-                console.log("importfromfile--", item);
-                const allChildren = await getAllChildren(dispatch, childAge, 1);
+                console.log("importfromfile--", item[0]);
+                const allChildren:any = await getAllChildren(dispatch, childAge, 1);
+                console.log('here log allChildren',allChildren)
                 let childId = await dataRealmCommon.getFilteredData<ConfigSettingsEntity>(ConfigSettingsSchema, "key='currentActiveChildId'");
                 this.closeImportedRealm();
-
-                if (allChildren.length > 0) {
+                console.log('here log childId',childId)

```

Comment:
Make sure logs with unnecessary or sensitive information do not go to production.